.github/workflows/auto-assign-issues-prs.yml

```diff
@@ -37,8 +41,15 @@ jobs:
           owner: ${{ github.repository_owner }}
           repo: ${{ github.event.repository.name }}
           pr_number: ${{ github.event.pull_request.number }}
-      - run: "echo 'Issue output: ${{ steps.get_linked_issue.outputs.data }}'"
-      - run: |
-          echo '${{ steps.get_linked_issue.outputs.data }}' | \
+      - name: Show the linked issue output
+        env: 
+          RETRIEVED_ISSUE: ${{ steps.get_linked_issue.outputs.data }}
+        run: "echo 'Issue output: $RETRIEVED_ISSUE'"
```

Comment:
Within single quotes the variable expansion is disabled :) (one of the things I tried to remember about shells are about variable expansion).